website/config.toml

```diff
@@ -143,3 +140,6 @@ pre = "<i class='fab fa-github'></i>"
   version = "v0.4.3"
   url = "https://karpenter.sh/v0.4.3-docs/"
 
+[[params.versions]]
+  version = "(Pre-release)"
+  url = "https://karpenter.sh/docs/"
```

Comment:
should we always have a "pre-release-docs" version going? we don't currently.

Reply:
I'm leaning towards enabling pre-release docs only as needed.

Reply:
i thought the docs in main branch is the pre-release :)) my bad if it's not what it is